### PR TITLE
Updated the phoneid object to include DELETE.

### PR DESCRIPTION
**Pull request by erikkai**
This is the first part of the changes for the Python SDK for the Consent API.

### CODE REVIEW

File: telesign/rest.py [hunk 1]
```diff
@@ -41,6 +41,8 @@ def __init__(self, requests_response):
             self.body = requests_response.text
             self.ok = requests_response.ok
 
+            print(requests_response.request.body)
```
Comment (ramsay) [inline telesign/rest.py]: remove this debug print

**Comment by ramsay** [comment]
Update the `version` in setup.py

File: telesign/rest.py (hunk 1 above)
Comment (erikkai) [reply telesign/rest.py]: okay added the changes

**Comment by ramsay** [comment]
Please fix the unit tests.

**Comment by adohertyTS** [comment]
DELETE is not currently a supported method for the Telesign Phone ID API and we also do not currently support a Consent API. Closing this out.

**Pull request closed without merging**